core/dbt/parser/manifest.py

```diff
@@ -170,9 +176,34 @@ def parse_project(
         # per-project cache.
         self._loaded_file_cache.clear()
 
+        project_info: Dict[str, Any] = {'parsers': []}
+        start_timer = time.perf_counter()
+        total_path_count = 0
         for parser in parsers:
+            parser_path_count = 0
+            parser_start_timer = time.perf_counter()
             for path in parser.search():
                 self.parse_with_cache(path, parser, old_results)
+                parser_path_count = parser_path_count + 1
+                if parser_path_count % 100 == 0:
+                    print("..", end='', flush=True)
```

Comment:
Is this for debugging or is this for the user? Can you use `logger.info` instead?

Reply:
It's one of those lines that show progress:
Running with dbt=0.19.0-b1
............................................................................
Found 1727 models, 8507 tests, 0 snapshots, 6 analyses, 1450 macros, 0 operations, 29 seed files, 283 sources

I can't keep it one line with logger.info, it would be:
Running with dbt=0.19.0-b1
..
..
..

instead.

I could remove it, do it with logger.info (though I don't like the way that looks particularly) or try to add it to the logger as a new method.

Reply:
Got it! This is fine for now. We can follow up if needed 👍